### PR TITLE
@sweir27 => [Search] Show artist series in search

### PR DESCRIPTION
**Pull request by mzikherman**
If the artist series data is included in search results (which is determined wholly by the gravity feature flag), then it will correctly render:

search

(The description, and proper image fallback should also appear shortly once gravity is deployed and artist series re-indexed).

### CODE REVIEW

File: src/v2/Apps/Search/Components/NavigationTabs.tsx [hunk 1]
```diff
@@ -57,7 +58,7 @@ export class NavigationTabs extends React.Component<Props> {
     const tabName = text.replace(/[0-9]/g, "").trim()
     return (
       <RouteTab
-        to={to}
+        to={to.replace(/\s/g, "_")}
```
Comment (sweir27) [inline src/v2/Apps/Search/Components/NavigationTabs.tsx:61]: Why was this change necessary?

File: src/v2/Apps/Search/routes.tsx [hunk 2]
```diff
@@ -19,6 +19,7 @@ const prepareVariables = (_params, { location }) => {
 }
 
 const tabsToEntitiesMap = {
+  artist_series: ["ARTIST_SERIES"],
```
Comment (sweir27) [inline src/v2/Apps/Search/routes.tsx:22]: It would be worth confirming with Brittney/Nicole that we want this to show as a separate tab (vs. under "More")!

File: src/v2/Apps/Search/Components/NavigationTabs.tsx (hunk 1 above)
Comment (mzikherman) [reply src/v2/Apps/Search/Components/NavigationTabs.tsx:61]: Removes whitespace from the tab (so /artist series -> /artist_series).

File: src/v2/Apps/Search/routes.tsx (hunk 2 above)
Comment (mzikherman) [reply src/v2/Apps/Search/routes.tsx:22]: It should be treated the way collections are (i believe), which has its own tab.

**Comment by mzikherman** [comment]
Merging this in!
